src/components/PersonInfo.tsx

```diff
@@ -0,0 +1,65 @@
+import { FC } from 'react';
+import { Person } from '../types';
+import { Link, useParams } from 'react-router-dom';
+import cn from 'classnames';
+
+interface Props {
+  person: Person;
+  people: Person[];
+}
+
+export const PersonInfo: FC<Props> = ({ person, people }) => {
+  const { sex, born, died, name, fatherName, motherName, slug } = person;
+  const { personSlug } = useParams();
+  const FEMALE = 'f';
+
+  const motherSlug = people.find(
+    ({ name: nameOfPerson }) => nameOfPerson === motherName,
+  );
+
+  const fatherSlug = people.find(
+    ({ name: nameOfPerson }) => nameOfPerson === fatherName,
+  );
```

Comment:
Would be better to prepare you array of `people` before render, and to add `mother` and `father` to every person object